src/utils/filter.ts

```diff
@@ -0,0 +1,23 @@
+import { Todo } from '../types/Todo';
+import { TodoFilter } from '../types/TodoFilter';
+
+export const filterTodos = (
```

Comment:
It looks good, but consider some ways to refactor this code (this function may be much shorter). Overall it is not criticall